src/pybind/mgr/dashboard/frontend/src/app/shared/components/copy2clipboard-button/copy2clipboard-button.component.html

```diff
@@ -0,0 +1,8 @@
+<button (click)="onClick()"
+        type="button"
+        class="btn btn-light"
+        i18n-title
+        title="Copy to Clipboard">
+<i [ngClass]="[icons.clipboard]">
+</i>
+</button>
```

Comment:
I am getting linting errors for this part locally. So I would suggest to make this as
```suggestion
<button (click)="onClick()"
        type="button"
        class="btn btn-light"
        i18n-title
        title="Copy to Clipboard">
  <i [ngClass]="[icons.clipboard]"></i>
</button>
```
Note that I added 2 empty spaces before `<i [ngClass]...` and brought the </i> in the same line as <i>

Reply:
fixed this as well!